### PR TITLE
Remove dependences of cv2

### PR DESCRIPTION
**Pull request by LielinJiang**
### PR types

Others
### PR changes

Others
### Describe

Remove dependences of cv2.

image

### CODE REVIEW

File: python/paddle/vision/transforms/functional.py [hunk 1]
```diff
@@ -214,18 +218,14 @@ def pad(img, padding, fill=(0, 0, 0), padding_mode='constant'):
...
-           center=None):
+def rotate(img, angle, interpolation=1, expand=False, center=None):
     """Rotates the image by angle.
 
     Args:
         img (numpy.ndarray): Image to be rotated.
         angle (float|int): In degrees clockwise order.
         interpolation (int, optional):
-            interpolation: Interpolation method.
+            interpolation: Interpolation method. Default: 1.
```
Comment (qingqing01) [inline python/paddle/vision/transforms/functional.py]: 227和228行重复。 详细的说明下不同int值的插值方式是什么

File: python/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
-opencv-python<=4.2.0.32
 requests>=2.20.0
```
Comment (qingqing01) [inline python/requirements.txt:1]: requirements中去掉后，CI的脚本里应该需要加下 https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/scripts/paddle_build.sh

File: python/paddle/vision/transforms/functional.py (hunk 1 above)
Comment (LielinJiang) [reply python/paddle/vision/transforms/functional.py]: done, thanks

File: python/requirements.txt (hunk 2 above)
Comment (LielinJiang) [reply python/requirements.txt:1]: 感谢，这个田硕老师帮忙解决了：https://github.com/PaddlePaddle/Paddle/pull/27378

**Review by qingqing01 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by saxon-zh — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]

**Pull request merged**